docs/Tutorial.md

```diff
@@ -80,20 +80,13 @@ const App = () => <Admin dataProvider={dataProvider} />;
 export default App;
 ```
 
-Also, remove the default Vite CSS fom the `main.tsx` file:
+Also, change the default Vite CSS file to look like this:
 
 ```diff
-// in src/main.tsx
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App'
--import './index.css'
-
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-)
+// in src/index.css
```

Comment:
There are two more differences between simple project's `index.html` and this tutorial's `index.html`

1- Script to adds the Roboto font
2- `<script>window.global = window</script>`

I don't know how relevant these are for a tutorial, let me know if I should add them to this PR